fv3core/stencils/ray_fast.py

```diff
@@ -30,18 +28,21 @@ def dm_layer(rf, dp, wind):
 
 
 @gtstencil()
-def dm_stencil(
-    dp: sd,
-    dm: sd,
-    pfull: sd,
-    rf: sd,
+def ray_fast_wind(
+    u: FloatField,
+    v: FloatField,
+    w: FloatField,
+    dp: FloatField,
+    pfull: FloatField,
     dt: float,
     ptop: float,
     rf_cutoff_nudge: float,
     ks: int,
+    hydrostatic: bool,
```

Comment:
Interestingly, I tried that, and `gt4py` gives an error because it cannot infer the type of `namelist.hydrostatic`,
```
E       AttributeError: 'BuiltinLiteral' object has no attribute 'data_type'
```
Shall we raise an issue?

Reply:
Great! @jdahm is working on a fix for this as we speak...